### PR TITLE
enh/write tests for model 37/40

### PR DESCRIPTION
**Pull request by ChrisKre**

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "python.pythonPath": "C:\\Users\\Home\\Anaconda3\\envs\\tensorflow\\python.exe"
```
Comment (frzifus) [inline .vscode/settings.json]: maybe gitignore?

File: .vscode/settings.json [hunk 2]
```diff
@@ -1,3 +1,3 @@
 {
-    "python.pythonPath": "C:\\Users\\Home\\Anaconda3\\envs\\tensorflow\\python.exe"
+    "python.pythonPath": "C:\\Users\\ckrem\\Anaconda2\\envs\\tensorflow\\python.exe"
```
Comment (frzifus) [inline .vscode/settings.json]: there's still some hard codet paths left.

File: models/Trainer.py [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+from keras import callbacks
+from optimizer.cb import Cback
+import json
+
+class Trainer:
+	
+	def __init__(self, model, filepath, identifier):		
+		self.filepath = filepath
+		self.model = model		
+		self.saved_model_path = 'C:\\Users\\ckrem\\Desktop\\IP'
```
Comment (frzifus) [inline models/Trainer.py]: and here

File: models/test.py [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+from keras.optimizers import SGD
+from keras.backend import eval
+import numpy as np
+from optimizer.LR_SGD import LR_SGD
+from OurModel import OurModel
+from keras.legacy import interfaces
+from DataLoader import DataLoader
+from Trainer import Trainer
+
+filepath = 'C:\\Users\\ckrem\\Desktop\\IP\\age.tfrecords '
```
Comment (frzifus) [inline models/test.py]: wieso keine relativen pfade?
```python
os.getcwd() # returns the current path.
```

Comment (ChrisKre) [reply models/test.py]: It was just for testing issues. We haven't had specified a structure where to put the tfrecords at that moment. Thanks for the advice with os.getcwd() :)

**Pull request merged**